packages/salesforcedx-vscode-lwc/src/commands/forceLightningLwcPreview.ts

```diff
@@ -70,6 +71,12 @@ interface PreviewQuickPickItem extends vscode.QuickPickItem {
   platformName: keyof typeof PlatformName;
 }
 
+export interface DeviceQuickPickItem extends vscode.QuickPickItem {
+  label: string;
```

Comment:
You don't need the properties that already exist in `QuickPickItem`. Just anything additional (i.e. `name`).

Reply:
🤦  classic newbie